modules/tensor_mechanics/include/materials/RadialReturnStressUpdate.h

```diff
@@ -68,6 +69,10 @@ class RadialReturnStressUpdate : public StressUpdateBase
 
   MaterialProperty<Real> & _effective_inelastic_strain;
   MaterialProperty<Real> & _effective_inelastic_strain_old;
+  Real _max_inelastic_incr;
+
+private:
+  Real _max_inc;
```

Comment:
Just because I see these comments often, I want to say that I disagree that protected should be the default accessor mode in the MOOSE (and the modules). Private accessors ensure object integrity which goes out the window once you go protected. WIth a code like MOOSE where every object is inherited, there's nothing wrong with considering which objects belong in private and which belong in protected. Protected with an inheritance heavy code gives many objects free reign to mess with base class state where it may be difficult to understand all of the corner case ways that an object's integrity can be compromised.